apps/src/javalab/Playground.js

```diff
@@ -31,6 +31,11 @@ export default class Playground {
     );
   }
 
+  onStop() {
```

Comment:
Might be out of the scope of this PR, but I think we'll have to update some state here too - we'd probably need to set `this.isGameOver = true` and `this.isRunning = false`? I believe @molly-moen was working on stopping the lambda function on stop which should also prevent us from getting updates after stop is called

Reply:
Actually looking at the code, looks like we do close the connection so we should stop receiving updates, and when we call run again, we reset the state anyway. Might be good to set those two values just in case, but I don't think it should affect user experience.

Reply:
Yeah making onStop() work generally here feels out of scope to me

Reply:
I can add though I guess? Seems straightforward enough

Reply:
Yeah up to you, I don't think it has to be part of this change since it shouldn't affect functionality - we can log it as a followup